src/index.js

```diff
@@ -103,3 +102,98 @@ const generateTasks = () => {
 };
 orderTasks();
 generateTasks();
+
+// start addRemove js
+
+const addNewTask = () => {
+  const inputAdd = document.querySelector('#tasks-text').value;
+  let newIndex;
+  if (tasks === null) {
+    newIndex = 1;
+  } else {
+    newIndex = tasks.length + 1;
+  }
+  const task = {
+    description: inputAdd,
+    index: newIndex,
+    completed: false,
+  };
+  tasks.push(task);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  return task;
+};
+const generateOneTask = (task) => {
+  const div1 = document.createElement('div');
+  const div2 = document.createElement('div');
+  const squareIcon = document.createElement('input');
+  squareIcon.type = 'checkbox';
+  const listText = document.createElement('input');
+  listText.type = 'text';
+  const threeDotIcon = document.createElement('i');
+  div1.appendChild(div2);
+  div1.appendChild(threeDotIcon);
+  div2.appendChild(squareIcon);
+  div2.appendChild(listText);
+
+  div1.className = 'list-items';
+  div2.className = 'left-items';
+  listText.className = 'list-text1';
+  squareIcon.className = 'checkbox';
+  threeDotIcon.className = 'fas fa-ellipsis-v three-dot-icon';
+  document.querySelector('.box').appendChild(div1);
+  listText.value = task.description;
+
+  const deleteIcon = document.createElement('i');
+  deleteIcon.className = 'fas fa-trash';
+  listText.addEventListener('keyup', () => {
+    task.description = listText.value;
+    threeDotIcon.replaceWith(deleteIcon);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  });
```

Comment:
- [ ] I could not find the button or link for deleting one task, would you add it?

Reply:
Hello Mugirase Emmanuel, Thank you for mentioning this. I added the button inside an addEventListener and you can see in the image below
![tse remark](https://user-images.githubusercontent.com/60292761/130329552-4f4f4497-e96e-41b1-b3af-860f1bbcfcbb.PNG)